### PR TITLE
Feature/add adaptive to screens

### PR DESCRIPTION
**Pull request by sergeyevalex90**
Adaptive for Login page
Adaptive for Registration page
Some visual changes of the calendar
Adaptive for Calendar page
Mobile header with burger menu button
Mobile sidebar

### CODE REVIEW

File: front-end/src/components/common/MobileHeader/MobileHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import PropTypes from 'prop-types';
+
+// Context
+import { CurrentUserContext } from 'context/AppProvider';
+
+// Assets
+import logo1 from 'assets/sidebar/logo-letter.png';
```
Comment (andriydzhrvts) [inline front-end/src/components/common/MobileHeader/MobileHeader.jsx]: Why logo1? Rename to logo

File: front-end/src/components/common/MobileHeader/MobileHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+const MobileHeader = ({ showSidebar, sidebarState }) => {
+  const { currentUser } = useContext(CurrentUserContext);
+
+  const handleClick = () => {
+    showSidebar(!sidebarState);
+  };
+
+  const userNameArr = currentUser.fullName.split(' ');
+  const cuttedUserName = `${userNameArr[0]} ${userNameArr[1][0]}.`;
```
Comment (andriydzhrvts) [inline front-end/src/components/common/MobileHeader/MobileHeader.jsx:25]: Is `userNameArr` matrix? What mean `userNameArr[1][0]`?

Comment (sergeyevalex90) [reply front-end/src/components/common/MobileHeader/MobileHeader.jsx:25]: I use userNameArr for splitting strings like this "Alex Serhieiev", and then I make this string to this "Alex S." for mobile version

Comment (andriydzhrvts) [reply front-end/src/components/common/MobileHeader/MobileHeader.jsx:25]: Got it. Thanks!

**Review by andriydzhrvts — APPROVED** [review]
Approve after resolve comments.

File: front-end/src/components/common/MobileHeader/MobileHeader.jsx (hunk 1 above)
Comment (sergeyevalex90) [reply front-end/src/components/common/MobileHeader/MobileHeader.jsx]: Done

**Pull request merged**